core/services/validators.go

```diff
@@ -103,6 +104,30 @@ func ValidateInitiator(i models.Initiator, j models.JobSpec) error {
 	}
 }
 
+func validateRunLogInitiator(i models.Initiator, j models.JobSpec) error {
```

Comment:
Given your explanation about why we need to do this now, I think I'd either make a note here in a comment, or in the new story in Tracker you created, indicating that this is temporary code that we intend to remove once PendingConfirmations isn't being overloaded anymore.

Reply:
These validations will continue to live on, but they will get more strict. Instead of erroring on `ethTxCount > 1` we will error on `ethTxCount != 1`. Once we tighten that, the related integration test with a minimal Job Spec starts failing.